server/vitals.js

```diff
@@ -140,8 +140,34 @@ async function checkButtonBatteries() {
   }
 }
 
+function logClientMessage(clientMessages) {
+  // Check that clientMessages contains data
+  if (Object.keys(clientMessages).length > 0) {
```

Comment:
If `clientMessages` is empty (`{}`), then `Object.values(clientMessages)` will return an empty array: `[]`. `forEach` on an empty array simply won't iterate, so I think this outer check for if there are no keys in `clientMessages` is not needed.

Reply:
This shouldn't be an issue because if `clientMessages` is empty then it wouldn't even hit the line with `Object.values(clientMessages)` because doesn't pass the conditional `if (Object.keys(clientMessages).length > 0)`. So the function will just be called and nothing will happen.

Reply:
It's not an issue; I'm just saying that the outer check for `Object.keys(clientMessages).length > 0` is unnecessary, as the inner `forEach` will do nothing if `clientMessages` is empty.

Reply:
So the function can be simplified :)

Reply:
Ahh I misunderstood but I see what you mean now! That's a very good point, I'll take out the conditional statement!